src/Chapter1.hs

```diff
@@ -652,8 +662,8 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = if (div n 10) > 9 then (firstDigit (div n 10)) else div n 10
```

Comment:
I see, that for a negative number like -19, this implementation will not work correctly in here 🤔 


Also, You can see that `div n 10` is reused several times in this function. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼